privacy-policy.md

```diff
@@ -0,0 +1,131 @@
+# Privacy Policy
+
+**Effective Date:** [Insert Date]
+
+The Python Asia Organization (PAO) is committed to protecting the privacy and personal data of its members, website visitors, and stakeholders. This Privacy Policy outlines how we collect, use, and safeguard personal information in compliance with the General Data Protection Regulation (GDPR), Estonian data protection laws, and privacy regulations across East and Southeast Asia.
+
+## 1. Who We Are
+
+The Python Asia Organization (PAO) is a non-profit entity registered in Estonia, dedicated to promoting the Python programming language in East and Southeast Asia. Our registered address is [Insert Address]. For privacy-related inquiries, please contact us at [privacy@pythonasia.org](mailto:privacy@pythonasia.org).
```

Comment:
I think this address is Estonia's. @iqbalabd Do you think it?

Reply:
So maybe this is a good time to set some guidance on how to use our addresses.

1. Our "mailing address" where people send us things should be our Tokyo's address
2. Our "business address" where we physically operate should be our Tokyo's address
3. Our "registered address" can be our Estonia's address.

Most of the time, I think we will use "mailing" and "business" addresses. But there are certain times like when making a bank account like on Wise or when we need to apply for something in the EU or Estonia which we will then use our Estonia's registered address. 

Anything official from the Estonian government will go to our Estonian address and Magrat will receive them, but if someone sends us t-shirts or cookies to our Estonian address, Magrat *will not* take it for us.

Reply:
This section "Our registered address is [Insert Address]." I think we need to use Estonia's address, Is it rigth?